scss/_patterns_notifications.scss

```diff
@@ -1,62 +1,146 @@
-        padding: 0 0 0 2 * $sph-inner;
-      }
+      margin-top: 0;
+      padding: 0 0 0 2 * $sph-inner;
 
       &::before {
         display: none;
       }
     }
 
     &.is-inline {
-      .p-notification__title {
-        display: inline;
+      .p-notification__content {
+        margin-bottom: $notification-text-margin-bottom;
+        padding-top: map-get($nudges, nudge--p) + map-get($browser-rounding-compensations, h5);
       }
 
+      .p-notification__title,
       .p-notification__message {
         display: inline;
+        margin: 0;
+        padding: 0;
+      }
+    }
+
+    &.is-modal {
+      box-shadow: $box-shadow;
+
+      .p-notification__actions:only-child {
+        margin-left: unset;
+      }
+
+      &::before {
+        display: none;
+      }
+    }
+
+    &.is-raised {
+      box-shadow: 0 0 2rem 0 rgba($color-x-dark, $shadow-opacity);
+    }
+
+    &.is-unread {
+      .p-notification__title {
+        position: relative;
+
+        &::after {
+          @include vf-icon-status-small($color-link);
+          @include vf-icon-size($default-icon-size);
+
+          content: '';
+          position: absolute;
+          top: $notification-icon-vert-offset + map-get($nudges, nudge--small);
+        }
       }
     }
   }
 
   .p-notification__content {
-    background-position: $sph-inner $notification-content-vert-padding + $notification-icon-vert-offset;
-    background-repeat: no-repeat;
-    background-size: map-get($icon-sizes, default);
-    margin-bottom: 0;
-    margin-top: -1px;
-    padding: $notification-content-vert-padding $sph-inner + $spv-inner--small $notification-content-vert-padding $notification-content-icon-space;
+    padding-right: 2 * $sph-inner;
   }
 
   .p-notification__title {
-    @extend %bold;
+    @extend %vf-heading-5;
 
-    display: block;
+    margin-bottom: 0.1rem;
   }
 
   .p-notification__message {
-    display: block;
+    @extend %default-text;
+
+    margin-bottom: $notification-text-margin-bottom;
+  }
+
+  .p-notification__close {
+    @extend %vf-hide-text;
+    @include vf-icon-close($color-mid-dark);
+    @include vf-icon-size($default-icon-size);
+
+    background-color: transparent;
+    background-position: center;
+    background-repeat: no-repeat;
+    border: 0;
+    margin: 0;
+    padding: $sp-unit;
+    position: absolute;
+    right: $sph-inner;
+    top: $notification-content-vert-padding + $notification-icon-vert-offset;
+  }
+
+  .p-notification__meta {
+    display: flex;
+    justify-content: space-between;
+    padding-right: $sph-inner;
+  }
+
+  .p-notification__time {
+    @extend %default-text;
+    @extend %muted-text;
+
+    margin-bottom: $notification-text-margin-bottom;
+  }
+
+  .p-notification__actions {
+    display: flex;
+    margin-bottom: $notification-text-margin-bottom;
+
+    &:only-child {
+      margin-left: auto;
+    }
+  }
+
+  .p-notification__action {
+    margin: 0;
+
+    [class*='p-button'] {
```

Comment:
Is this needed? It seems that action buttons have `p-notification__action` class which resets margin to 0 already.

Reply:
This was just me trying to get contextual menus to work in the notification, which has a button as a child of the `.p-contextual-menu` component. But since we're not doing the notification center stuff just yet I'll remove this